src/screens/SearchScreen/components/SearchBar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {View, TextInput, Button} from 'react-native';
+import PropTypes from 'prop-types';
+
+const PLACEHOLDER_TEXT = 'What word do you want to look up?';
+
+const SearchBar = props => {
```

Comment:
We commonly destructure props heres to save on typing later on, but it's personal preference. If you like seeing `props.updateData` and `props.setSearchTerm` below instead of just `searchTerm` that's a valid position to have. My preference is to not see and read `props.` repeatedly within the function, but again I could see a case where it is actually nice to have the context that "this comes from props" in the case of a lengthier more complicated component.